source/includes/sharding-warning.rst

```diff
@@ -1,13 +1,13 @@
 
-.. warning::
-
-   Syncing large documents to a sharded destination cluster can cause 
-   the destination cluster to initiate chunk migration.  When the destination
-   cluster migrates chunks during sync, it may trigger a bug that can result
-   in data loss.
-
-   To avoid this, it is recommended that you run the :dbcommand:`balancerStop`
-   command on the destination cluster before starting a sync.  Once ``mongosync``
-   completes the sync, you can restart the balancer with the 
-   :dbcommand:`balancerStart` command.
-
+.. important::
+
+   If the destination is a sharded cluster with the balancer enabled, write 
+   performance of the destination cluster may be degraded during ``mongosync``'s 
```

Comment:
[style guide] Per the writing guide,  could we make this [active voice](https://www.mongodb.com/docs/meta/style-guide/writing/use-active-voice/)? 

For ex, "write performance of the destination cluster may decline during ..."

Reply:
👍  but kept "degrade" over "decline".